contributing.md

```diff
@@ -66,7 +66,9 @@
  * Leave a comment
    * If you are solving an issue (e.g. ```#17```), add ```Closes #17``` in the comment, the issue will automatically be closed when the pull request is merged.
 
-
+### 7. Delete branch locally and/or remotely after pull request is merged on GitHub
+ * From the command line: ```git branch -d new-branch```
+ * Additionally if you want to delete your remote branch: ```git push origin : new-branch```
```

Comment:
not sure if you need the 3 backticks, I think with one should be all right. Also, probably a blank line between the end of this paragraph and the title of the next section could be good.

Reply:
@datapythonista I cleaned up the file. Thank you!

Reply:
@sparalic  do you think git branch -d **branch-to-be-deleted**  could be a better alternative in this context, rather than **new-branch** ?